Resources/Plists/PreferenceDefaults.plist

```diff
@@ -101,6 +101,8 @@
 	<integer>20</integer>
 	<key>EditInSheetForMultiLineText</key>
 	<true/>
+	<key>SaveApplicationUsageAnalytics</key>
+	<true/>
```

Comment:
Makong sure - this means ON by default, right?

Reply:
That is correct